engine/src/flutter/lib/ui/window.dart

```diff
@@ -968,6 +969,20 @@ class AccessibilityFeatures {
   /// Only supported on iOS.
   bool get onOffSwitchLabels => _kOnOffSwitchLabelsIndex & _index != 0;
 
+  /// Whether accessibility announcements (like [SemanticsService.announce])
+  /// are allowed on the current platform.
+  ///
+  /// Returns `false` on Android, where platform announcements are deprecated
+  /// by the underlying platform.
+  ///
+  /// Returns `true` on all other platforms (iOS, web, desktop) where such
+  /// announcements are generally supported without discouragement.
+  ///
```

Comment:
This is the most specific use of platforms I would encourage. They are given as examples and even those could be wrong with time but it is better than promising something that we don't know. 
```suggestion
  /// Returns `true` on platforms where such announcements are
  ///  generally supported without discouragement. (iOS, web etc) 
  ///
```